### PR TITLE
Made search use quality.

### PR DESCRIPTION
**Pull request by AlexGilleran**

### CODE REVIEW

File: lerna.json [hunk 1]
```diff
@@ -30,7 +30,8 @@
   ],
   "version": "0.0.28-SNAPSHOT",
   "hoist": [
-    "tsmonad"
+    "tsmonad",
+    "@types/express"
```
Comment (kring) [inline lerna.json]: Are you sure this is necessary?  I can't think of why it would be.  😟

**Comment by kring** [comment]
This looks great to me, once the tests are behaving.

File: lerna.json (hunk 1 above)
Comment (AlexGilleran) [inline lerna.json]: oh no that was just to make it work before 😬

**Comment by AlexGilleran** [comment]
Success!

**Pull request merged**